portal/migrations/versions/908731b85112_.py

```diff
@@ -0,0 +1,85 @@
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+
+from portal.models.questionnaire_response import QuestionnaireResponse
+
+
+"""empty message
```

Comment:
If you put the comment block at the top, with "empty message" filled in, it'll show when running the upgrade.
We should probably reorder the template - I just haven't taken the 12 seconds to do so...